### PR TITLE
Register stro.nikhef.nl as a submit node.

### PR DESCRIPTION
**Pull request by hestem**
This resource group will add the Virgo Submit infrastructure to the OSG topology.

### CODE REVIEW

**Comment by efajardo** [comment]
@hestem , I registered you into the contact database. For the id use:

Group_id: 517
Resource_id: 1027

File: Virgo-submit-infrastructure.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+  
+Production: false
+SupportCenter: Self Supported
+
+GroupDescription: This is a cluster of submission hosts for the IGWN. 
+GroupID: <ID>
```
Comment (efajardo) [inline Virgo-submit-infrastructure.yaml]: Please use:

```suggestion
GroupID: 517
```

File: Virgo-submit-infrastructure.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Resources:
+  Virgo-submit-infrastructure:
+    Active: false
+    Description: stro.nikhef.nl is a glidein frontend node. 
+    ID: <ID>
```
Comment (efajardo) [inline Virgo-submit-infrastructure.yaml]: Please us resource id:
```suggestion
ID: 1027
```

**Comment by hestem** [comment]
> @hestem
> 
> The file was added in the wrong place try adding it to
> 
> ```
> Dutch National e-Infrastructure/NIKHEF/Virgo-submit-infrastructure.yaml
> ```

ahh sorry about that. You're totally right. NIKHEF should be Nikhef though.

File: topology/Dutch National e-Infrastructure/Nikhef/Virgo-submit-infrastructure.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+  
+Production: false
+SupportCenter: Self Supported
+
+GroupDescription: This is a cluster of submission hosts for the IGWN. 
+GroupID: <ID>
```
Comment (efajardo) [inline topology/Dutch National e-Infrastructure/Nikhef/Virgo-submit-infrastructure.yaml]:
```suggestion
GroupID: 517
```

File: topology/Dutch National e-Infrastructure/Nikhef/Virgo-submit-infrastructure.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Resources:
+  Virgo-submit-infrastructure:
+    Active: false
+    Description: stro.nikhef.nl is a glidein frontend node. 
+    ID: <ID>
```
Comment (efajardo) [inline topology/Dutch National e-Infrastructure/Nikhef/Virgo-submit-infrastructure.yaml]:
```suggestion
    ID: 1027
```

**Pull request merged**